internal/providers/terraform/azure/virtual_network_gateway_connection_test.go

```diff
@@ -0,0 +1,17 @@
+package azure_test
```

Comment:
ignore this file it was from renaming `vpn_gateway_connection_test` (see PR comment)